test/test_institutions.rb

```diff
@@ -1,5 +1,8 @@
 require_relative 'test_helper'
 
+# Using strict models to catch any new fields returned by the API
+Plaid.relaxed_models = true
+
```

Comment:
If you wanted to catch new fields returned by the API you would want to to set this to `false`, which it should already be set to when running tests (see `test/test_helper.rb:23`)

Reply:
Ideally you never would have to set this in an individual tests and the tests should pass with `relaxed_models = false`

Reply:
Oh oops I forgot to update the comment here, but basically we have some undocumented flexible input fields that we've added to the institutions endpoints that we don't want externally exposed or documented yet.

They should only ever be used internally to determine how to login to an institution and we don't want clients to use them to try and do a direct integration or anything. Because of that I didn't include those fields in the `Institution` model and the test was failing because it didn't know how to coerce those fields.